spec/index.bs

```diff
@@ -8329,6 +8334,9 @@ Only formats with {{GPUTextureSampleType}} {{GPUTextureSampleType/"float"}} can
 The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}}
 usage in the core API, including both {{GPUStorageTextureAccess/"read-only"}} and {{GPUStorageTextureAccess/"write-only"}}.
 
+The {{GPUTextureUsage/STORAGE|GPUTextureUsage.RENDER_ATTACHMENT}} column specifies the support for
```

Comment:
Do you think this is helpful? "The A column specifies the support for A usage in the core API".
Just asking about the motivation here: are you trying to make it consistent with all the columns described, or you believe this makes it clearer for some people.

Reply:
I do think it is clearer to add it. See the text in the subsection `Plain color formats`, we talked all usages (COPY_SRC, COPY_DST, SAMPLED, and STORAGE), but we didn't mention RENDER_ATTACHMENT via any text. You know, most plain color formats can support RENDER_ATTACHMENT. It might be obvious in the table, but I think it is better to state this clearly in specification. Don't let readers to guess.